### DISCUSSION TITLE
[HELP] Can't Try Maui

### DISCUSSION THREAD

**Discussion by Thefrenchdev** [dotnet/maui#2362] [Q&A] [answered]
Hi, 

I'm trying to use the Maui Preview and keep failing so trying here to get some help.

I've installed vs2022 preview with all required components following the net maui docs.
the Maui-check keep saying that everythings is OK But Still not working 
Here are the issues : 

![errors_Maui](https://user-images.githubusercontent.com/70511981/131818742-9e9029d2-e7c8-4329-a984-ffa6d6c3df53.png)

I've tried lot of things and above all "dotnet worlkoads install maui" and stil not OK

**Comment by acaliaro** [comment, marked as answer]
have you tried to specifiy the package?
dotnet workloads install maui-ios.....

**Reply by Thefrenchdev** [reply]
Thx but Already tried

**Comment by juansturla** [comment]
It is weird, but what worked for me is executing VS 2022 as Administrator.
Don't know why

**Reply by Thefrenchdev** [reply]
Didn't Work neither
I'm trying to uninstall everything and reinstall it.

**Reply by juansturla** [reply]
You should able to see the installed workloads with `dotnet workload list`
![image](https://user-images.githubusercontent.com/10248860/132199499-119cf1d0-3179-4518-bad4-4c9f7195855e.png)

**Reply by Thefrenchdev** [reply]
It Was empty!!

As I reinstalled everything i did not installed workload  manually.

I dit it and... It worked Fine!!!

Thank you

**Reply by acaliaro** [reply]
@Thefrenchdev can you explain what have you done? What have you reinstalled?
Vs2022?
Maui-check?
Thanks
